v3/plugins/che-incubator/theia-dev/0.0.1/meta.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       protocol: http
   containers:
    - name: theia-dev
-     image: "docker.io/eclipse/che-theia-dev:nightly"
+     image: "docker.io/eclipse/che-theia-dev:next"
```

Comment:
is this nightly / next switch intentional? what is the reason for changing it?

Reply:
@ibuziuk yes it's intentional, I've used a separate commit for it
https://github.com/eclipse/che-plugin-registry/pull/306/commits/b4860a53846b2830f83754cd819fe111339847e4

nightly was not built since 9months: https://hub.docker.com/r/eclipse/che-theia-dev/tags?page=1&name=nightly

as it was switched to next